tests/sentry/workflow_engine/handlers/action/test_action_handlers.py

```diff
@@ -20,7 +21,7 @@ def setUp(self):
     def test_execute_without_group_type(self):
         """Test that execute does nothing when detector has no group_type"""
         self.detector.type = ""
-        self.action.trigger(self.event_data, self.detector)
+        self.action.trigger(self.event_data, self.detector, str(uuid4()))
```

Comment:
this uuid wasn't being used, can we remove this stuff?

curious about the plan around this uuid as well, i wonder if this is something we should encapsulate inside of self.action.trigger rather than exposing it in the params